### ISSUE TITLE
ImportError: cannot import name current_app

### ISSUE THREAD

**Issue by juju4** [google/timesketch#245]
```
failed: [localhost] (item={u'p': u'bobpass', u'u': u'bob'}) => {"changed": true, "cmd": ["tsctl", "add_user", "-u", "bob", "-p", "bobpass"], "delta": "0:00:00.686229", "end": "2016-10-30 02:09:05.633392", "failed": true, "invocation": {"module_args": {"_raw_params": "tsctl add_user -u bob -p bobpass", "_uses_shell": false, "chdir": null, "creates": null, "executable": null, "removes": null, "warn": true}, "module_name": "command"}, "item": {"p": "bobpass", "u": "bob"}, "rc": 1, "start": "2016-10-30 02:09:04.947163", "stderr": "Traceback (most recent call last):\n  File \"/usr/local/bin/tsctl\", line 34, in <module>\n    from timesketch import create_app\n  File \"/usr/local/lib/python2.7/dist-packages/timesketch/__init__.py\", line 26, in <module>\n    from timesketch.api.v1.resources import AggregationResource\n  File \"/usr/local/lib/python2.7/dist-packages/timesketch/api/v1/resources.py\", line 39, in <module>\n    from flask_login import current_app\nImportError: cannot import name current_app", "stdout": "", "stdout_lines": [], "warnings": []}
```

both with xenial and trusty
https://travis-ci.org/juju4/ansible-timesketch/jobs/171709022#L1905
https://travis-ci.org/juju4/ansible-timesketch/jobs/171709023#L1632

It was fine 20d ago (without gift and ubuntu upstream plaso)
https://travis-ci.org/juju4/ansible-timesketch/jobs/166521531#L1776

**Comment by csash** [comment]
There was some refactoring in Flask-Login in the last 20 days that broke these imports. This is fixed in #244. If you need a workaround in the interim, downgrade to Flask-Login v0.3.2.

**Comment by juju4** [comment]
Thanks. that works!
